tests/test_TFUtil.py

```diff
@@ -1357,6 +1431,7 @@ def test_scatter_nd():
   session.run(ref_grad)
 
 
+@unittest.skip('v.copy_compatible_to(x) in rel_embed tries to match a [B,2,F|5] tensor to [B,T,F|5]. We do not allow that!')
```

Comment:
I wonder whether we should instead just remove the test case. We anyway have it in history. I'm not sure it is really helpful to keep. The test file is already too big.
(@JackTemaki or @patrick-wilken , any opinions?)

Also, I still wonder where this comes from originally. I assume I expected that this should work at some point. But I guess it's really much better if we disallow that (and the user can use `reinterpret_data` layer or so to enforce T=2 or whatever).

Reply:
I removed the test case for now, we can undo that commit again if @JackTemaki or @patrick-wilken think otherwise.